### ISSUE TITLE
No 

### ISSUE THREAD

**Issue by Jay991** [dotnet/core#6419]
I'm trying to compile the onlyoffice Community server code on an Ubuntu droplet. After installing .NET 5.0 with runtime, I'm having difficult installing the .NET Developer pack version 4.8. As the the below shows that the pack is much needed:

`# dotnet run 
/usr/share/dotnet/sdk/5.0.301/Microsoft.Common.CurrentVersion.targets(1216,5): error MSB3644: The reference assemblies for .NETFramework,Version=v4.8 were not found. To resolve this, install the Developer Pack (SDK/Targeting Pack) for this framework version or retarget your application. You can download .NET Framework Developer Packs at https://aka.ms/msbuild/developerpacks [/root/CommunityServer/common/ASC.Common/ASC.Common.csproj]

The build failed. Fix the build errors and run again.`

Where do I get a developer pack that is compatible with Ubuntu 20?

**Comment by mairaw** [comment]
I think this error message might be misleading. You can't run .NET Framework on Ubuntu. @rainersigwald any tips?

Can you run `dotnet --info` and paste the output here? I'm thinking you might be missing the .NET 5 SDK.

**Comment by rainersigwald** [comment]
> You can't run .NET Framework on Ubuntu.

That's true, though you should be able to _build_ for it, because the reference assembly package should be automatically referenced by the SDK since dotnet/sdk#10981.

That said, this error message is misleading. Filed https://github.com/dotnet/msbuild/issues/6647 to track.

I'm going to move this to the SDK repo since I think the reference should be added automatically by the SDK and that doesn't seem to be working here.
